instagram/src/App.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/PostContainer';
+import SearchBar from './components/SearchBar/SearchBar';
+import './App.scss';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      liked: false,
+      searchValue: '',
+      data: dummyData
+    };
+  }
+
+  liker = timestamp => {
+    this.setState({
+      liked: !this.state.liked
+    });
+
```

Comment:
I don't think you need `.slice()` when you're using `.filter()` because slice is basically copying the original array, but filter is already returning new array and it is not mutating the original array.